patches/chrome-browser-ui-views-frame-browser_view_layout.cc.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/chrome/browser/ui/views/frame/browser_view_layout.cc b/chrome/browser/ui/views/frame/browser_view_layout.cc
-index 7359b0457bf1734cc8383b7f169c7027ea84db9f..0232e7e7af4fdf607f68b9b424f1b50d5013910d 100644
+index 80dcf5b08c67c34b1932da03459e6311393f257c..77917a39bc0546d09c64b8a033074bd1ff620060 100644
```

Comment:
This change could be deleted.

Reply:
OK - I just rebased and squashed to help them 😄 This should fix the issue. PTAL!